deployments/jwebbinar/image/environments/setup-notebooks

```diff
@@ -13,6 +13,9 @@
 
 #/opt/common-scripts/set-notebook-kernel jwebbinar JWebbinar jwebbinar_prep/pipeline_products_session/*.ipynb
 
```

Comment:
Completely unrelated to this PR, but I've been thinking of requesting that we have text files in each jwebbinar_prep directory specifying which kernel should be used.

It's a pain to always be updating this.